Framework/Algorithms/src/CalculateFlatBackground.cpp

```diff
@@ -189,7 +189,7 @@ void CalculateFlatBackground::exec() {
             std::bind2nd(VectorHelper::AddVariance<double>(), variance));
       }
```

Comment:
It may be worth writing down places where you find this. Once we introduced `operator+` for our new types, we can then go back and replace this, without going through all the code once more.